src/corelibs/U2View/src/ov_msa/MSAEditor.cpp

```diff
@@ -788,4 +810,106 @@ void MSAEditor::sl_convertRawToAminoAlphabet() {
     msaObject->morphAlphabet(resultAlphabet);
 }
 
+void MSAEditor::sl_sortGroupsBySize() {
+    groupsSortOrder = sender() == sortGroupsBySizeAscendingAction ? GroupsSortOrder::Ascending : GroupsSortOrder::Descending;
+    updateCollapseModel();
+}
+
+// TODO: move this function into MSA?
```

Comment:
I understand, that this TODO wasn't added by you, but could you either explain the idea or just remove it? "move this function into MSA", but we are already in MSA, isnt it? Or what is this supposed to mean?

Reply:
I fixed this TODO and moved the method into MultipleAlignmentRowData